docker/Dockerfile

```diff
@@ -1,12 +1,14 @@
 FROM oraclelinux:8
 
 LABEL author="OPERA ADT" \
-    description="s1 cslc 0.2 Gamma release" \
-    version="0.2-gamma"
+    description="s1 cslc 0.3 Gamma release" \
+    version="0.3-gamma"
 
 RUN yum -y update &&\
     yum -y install curl git &&\
-    adduser compass_user
+    adduser compass_user &&\
+    mkdir -p /.config && chmod -R 777 /.config &&\
+    mkdir -p /.cache && chmod -R 777 /.cache
```

Comment:
Just out of curiosity why is this added?

Reply:
While testing Docker image, some dependencies complained about not having the writable directories as above, so I had to create them and give write permission.

Reply:
Can you add a comment to capture the above so this knowledge is not lost?